In [1]:
satisfaction = read.table("/home/isaac/R Projects, STAC67/A2, Q5")

In [2]:
satisfaction

V1 V2 V3 V4 
1  48 50 51 2.3
2  57 36 46 2.3
3  66 40 48 2.2
4  70 41 44 1.8
5  89 28 43 1.8
6  36 49 54 2.9
7  46 42 50 2.2
8  54 45 48 2.4
9  26 52 62 2.9
10 77 29 50 2.1
11 89 29 48 2.4
12 67 43 53 2.4
13 47 38 55 2.2
14 51 34 51 2.3
15 57 53 54 2.2
16 66 36 49 2.0
17 79 33 56 2.5
18 88 29 46 1.9
19 60 33 49 2.1
20 49 55 51 2.4
21 77 29 52 2.3
22 52 44 58 2.9
23 60 43 50 2.3
24 86 23 41 1.8
25 43 47 53 2.5
26 34 55 54 2.5
27 63 25 49 2.0
28 72 32 46 2.6
29 57 32 52 2.4
30 55 42 51 2.7
31 59 33 42 2.0
32 83 36 49 1.8
33 76 31 47 2.0
34 47 40 48 2.2
35 36 53 57 2.8
36 80 34 49 2.2
37 82 29 48 2.5
38 64 30 51 2.4
39 37 47 60 2.4
40 42 47 50 2.6
41 66 43 53 2.3
42 83 22 51 2.0
43 37 44 51 2.6
44 68 45 51 2.2
45 59 37 53 2.1
46 92 28 46 1.8

In [3]:
anxiety = satisfaction$V4

In [4]:
severity = satisfaction$V3

In [5]:
age = satisfaction$V2

In [6]:
satisfaction_var = satisfaction$V1

In [7]:
fit1 = lm(satisfaction_var~age+anxiety+severity, data = satisfaction)

In [8]:
x_0 = data.frame(age = 35, severity = 45, anxiety = 2.2)

In [9]:
predict(fit1, x_0)

1 
69.01029

CI for the mean of satisfaction_var (the Y variable) at x = x_0

In [11]:
predict(fit1, x_0, interval = "confidence", level = 0.9)

fit      lwr      upr     
1 69.01029 64.52854 73.49204

90% of satisfaction lies between 64.52854 and 73.49204 with an average of 69.01029

In [12]:
predict(fit1, x_0, interval = "prediction", level = 0.9)

fit      lwr      upr     
1 69.01029 51.50965 86.51092

We can expect the next value of patient's satisfaction to lie between 51.50965 and 86.51092.